src/modules/import/wmtsgetcap.js

```diff
@@ -49,8 +49,11 @@ exports = function(gettext, gettextCatalog, ngeoWmtsGetCapTemplateUrl) {
       }
 
       if (!layer['isInvalid']) {
-        const getTileMetadata = getCap['OperationsMetadata']['GetTile']['DCP']['HTTP']['Get'][0];
-        const requestEncoding = getTileMetadata['Constraint'][0]['AllowedValues']['Value'][0];
+        let requestEncoding;
+        if ('OperationsMetadata' in getCap && 'GetTile' in getCap['OperationsMetadata']) {
+          const getTileMetadata = getCap['OperationsMetadata']['GetTile']['DCP']['HTTP']['Get'][0];
+          requestEncoding = getTileMetadata['Constraint'][0]['AllowedValues']['Value'][0];
+        }
         const layerOptions = {
           'layer': layer['Identifier'],
           'requestEnconding': requestEncoding
```

Comment:
oups, the key name is `requestEnconding` but it should be `requestEncoding`

Reply:
Nice catch.